wsf_scraping/spiders/gov_spider.py

```diff
@@ -2,6 +2,7 @@
 from scrapy.http import Request
 from .base_spider import BaseSpider
```

Comment:
is `.base_spider` local? if so:

>>>
Imports should be grouped in the following order:

    Standard library imports.
    Related third party imports.
    Local application/library specific imports.

You should put a blank line between each group of imports.

Reply:
pep8 pedantry 🤓

Reply:
Well spotted 👍